testpilot/frontend/static-src/test/views/experiment-page.js

```diff
@@ -71,6 +72,29 @@ test('now active indicator shows when experiment is enabled', t => {
   t.equal(myView.query('.now-active').style.display, 'none');
 });
 
+test('introduction appears in view', t => {
+  t.plan(3);
+
+  const myView = new MyView({headerScroll: false, slug: 'slsk'});
+  const myModel = app.experiments.models[0];
+
+  myView.render();
+  const innerHTML = myView.query('[data-hook=introduction-html]').innerHTML;
+  const styleWhenNotEmpty = myView
+    .query('[data-hook=introduction-container]').style;
+  t.ok(innerHTML == myModel.introduction,
```

Comment:
nit: Possibly want `===` here.

Which also means we're either not linting the test/* directory, or maybe not checking for the `eqeqeq` ESLint rule.

**UPDATE:** Nope, definitely looks like we're not linting testpilot/frontend/static-src/test/**.